tox.ini

```diff
@@ -100,7 +96,8 @@ setenv =
     TWINE_USERNAME = {env:TWINE_USERNAME:__token__}
 commands =
     python -c "import path; path.Path('dist').rmtree_p()"
-    python -m pep517.build .
+    python -m build
+    python -m twine check --strict dist/*
```

Comment:
FYI, In jaraco/skeleton, the fidelity of the readme is checked prior to release (as part of the test suite run) using pytest-checkdocs.

Reply:
And if it's broken by a PR sent by someone it needs to be fixed in a new PR. Which can't be done prior to a release. So we're wasting release manager cycles by pushing this until then and not checking it in a PR

Reply:
I interpreted "prior to release" as "immediately prior to release", but I see now that @jaraco said "as part of the test suite run", i.e. via a typical `tox` run in CI.

pytest-checkdocs seems handy, but I think it's more important to dogfood `twine check`. Done in https://github.com/pypa/twine/pull/757/commits/802472a4f056e99e98952acfcf93cc319b12f05c.